lib/highlight.css

```diff
@@ -0,0 +1,199 @@
+  color: <<FORE_SECONDARY>>;
+}
+
+#loading-overlay.material paper-spinner .circle,
+#loading-overlay.material paper-spinner-lite .circle {
+  border-color: <<FORE_SECONDARY>><<NOTIMPORTANT>>;
+}
+
+a.primary {
+  color: <<FORE_SECONDARY>>;
+}
+
+paper-button.material-primary {
+  background-color: <<FORE_SECONDARY>>;
+}
+
+paper-checkbox #checkbox.checked {
+  border-color: <<FORE_SECONDARY>>;
+  background-color: <<FORE_SECONDARY>>;
+}
+
+paper-checkbox #ink {
+  color: <<FORE_SECONDARY>>;
+}
+
+paper-toggle-button[checked] .toggle-button {
+  background-color: <<FORE_SECONDARY>>;
+}
+
+paper-toggle-button[checked] .toggle-bar {
+  background-color: <<FORE_SECONDARY>>;
+}
+
+paper-toggle-button[checked] .toggle-ink {
+  color: <<FORE_SECONDARY>>;
+}
+
+.paper-slider-0 #sliderKnobInner.paper-slider,
+#sliderKnobInner.paper-slider,
+.paper-progress-0 #primaryProgress.paper-progress {
+  background-color: <<FORE_SECONDARY>>;
+}
+
+.nav-item-container.selected iron-icon {
+  color: <<FORE_SECONDARY>>;
+}
+
+.upload-dialog-title {
+  background-color: <<FORE_SECONDARY>>;
+}
+
+.paper-input-container-0 .focused-line.paper-input-container {
+  background-color: <<FORE_SECONDARY>>;
+}
+
+.rating-container li {
+  -webkit-filter: invert(1);
+}
+
+#player.material .material-player-middle paper-icon-button[data-id="repeat"][value="LIST_REPEAT"],
+#player.material .material-player-middle paper-icon-button[data-id="repeat"][value="SINGLE_REPEAT"],
+#player.material .material-player-middle paper-icon-button[data-id="shuffle"][value="ALL_SHUFFLE"],
+#player.material .material-player-middle paper-icon-button[data-id="repeat"][value="LIST_REPEAT"],
+#player.material .material-player-middle paper-icon-button[data-id="repeat"][value="SINGLE_REPEAT"],
+#player.material .material-player-middle paper-icon-button[data-id="shuffle"][value="ALL_SHUFFLE"] {
+  color: <<FORE_SECONDARY>>;
+}
+
+.cluster .lane-button core-icon,
+.cluster .lane-button iron-icon,
+#player.material .material-player-middle paper-icon-button[data-id="repeat"].active,
+#player.material .material-player-middle paper-icon-button[data-id="shuffle"].active,
+#player.material #material-player-right-wrapper paper-icon-button[data-id="queue"].opened {
+  color: <<FORE_SECONDARY>>;
+}
+
+.song-row .explicit,
+.material-card .explicit {
+  background-color: <<FORE_SECONDARY>>;
+}
+
+.material-detail-view .material-container-details .read-more-button {
+  color: <<FORE_SECONDARY>>;
+}
+
+.goog-buttonset-default {
+  background-color: <<FORE_SECONDARY>>;
+}
+
+button.goog-buttonset-default,
+button.goog-buttonset-default:hover {
+  background-color: <<FORE_SECONDARY>>;
+}
+
+.top-charts-view .song-row [data-col="index"] .column-content,
+.material-card .details .left-items .index,
+.more-songs-container {
+  color: <<FORE_SECONDARY>>;
+}
+
+/*
+* Highlight only mode specific things
+* These are prefixed here with ".gmusic-theme-highlight"
+*/
+
+.gmusic-theme-highlight #material-app-bar {
```

Comment:
I think we talked past each other again, after we move most of the CSS in here to a `common.css` and only show `highlight.css` when someone triggers it, then we should no longer need the classes:

https://github.com/gmusic-utils/gmusic-theme.js/pull/17#issuecomment-199137751